app/models/wiki.rb

```diff
@@ -0,0 +1,5 @@
+class Wiki < ActiveRecord::Base
+  belongs_to :user
+
+  scope :most_recently_updated, -> { order('updated_at DESC').limit(5)}
```

Comment:
Updated this scope to retrieve the 5 most recently changed wikis.  Will probably remove the limit(5) in favor of pagination later.